ports/mimxrt/sdcard.c

```diff
@@ -0,0 +1,963 @@
+#include "sdcard.h"
```

Comment:
What is the licensing and copyright of this file?

Reply:
copyright added to file. fixed